src/content/blog/ddev-dec-2024-newsletter.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "DDEV December 2024 Newsletter"
+pubDate: 2024-12-03
+#modifiedDate: 2024-09-06
+summary: DDEV November 2024 Newsletter
+author: Randy Fay
+featureImage:
+  src: /img/blog/2024/12/nancy_lewis_colorado_river.jpg
+  alt: Nancy Lewis painting of Colorado River near Palisade, Colorado
+categories:
+  - Community
+---
+
+**Hello DDEV Community!** It’s always a pleasure to reach out to you at the beginning of the month. Here are some updates for you.
+
+**DDEV v1.24.0** was released with lots of goodies, including the full PHP 8.4.1 with all extensions, including Xdebug. This is a major release with some deprecations and removals, please read the [release blog](release-v1.24.0-php8.4.md) and [release notes](https://github.com/ddev/ddev/releases/tag/v1.24.0).
+
+**TYPO3 Community Budget Ideas:** DDEV has posted a community budget idea for TYPO3's [community budget process](https://typo3.org/article/budget-2025-ideas-for-quarter-1-2025-published-vote-now). We'd like to improve DDEV's xhprof and xhgui even beyond where [@tyler36](https://github.com/tyler36) has already taken the [DDEV xhgui add-on](https://github.com/ddev/ddev-xhgui) and incorporate it into core. If you're a member of the TYPO3 association you've gotten an email about voting on this, please vote for the DDEV proposal!
+
+**DDEV Blogs and News**
+
+* [DDEV Usage Statistics](stats-on-ddev-usage-nov-2024.md)
+* [Setting up a Windows Machine for DDEV](windows-ddev-setup.md)
+* [Contributor Training: DDEV Analytics with Amplitude](https://ddev.com/blog/amplitude-ddev-analytics-contributor-training/)
+* [DDEV Advisory Group Meeting](https://github.com/orgs/ddev/discussions/6682). These meetings happen bimonthly and you're always invited. The next one is Wednesday, January 8, 2025, and is the most important meeting of the year. We will review 2024 and plan for 2025. We'd love to have you [join us](https://www.meetup.com/ddev-events/events/303197425).
```

Comment:
```suggestion
* [DDEV Advisory Group Meeting](https://github.com/orgs/ddev/discussions/6682). These meetings happen every two months and you're always invited. The next one is Wednesday, January 8, 2025, and is the most important meeting of the year. We will review 2024 and plan for 2025. We'd love to have you [join us](https://www.meetup.com/ddev-events/events/303197425).
```

"bimonthly" means "twice a month or every two months" so we should be more specific.